third-party/test/index.ts

```diff
@@ -71,6 +70,8 @@ export function defaultMochaOptions(useColors: boolean = true): MochaOptions {
             }
         }
     }
+    ;(o as any).color = useColors
```

Comment:
workaround because `@types/mocha` version in `package.json` is wrong. Don't know how to fix it